icecreamwars/icecreamwars/src/Components/Votes/Votes.css

```diff
@@ -0,0 +1,48 @@
+h2 {
+  font-family: "Caveat", sans-serif;
+  font-size: 3rem;
+}
+
+.vote-section {
```

Comment:
Your positioning here is keeping this votes section on the right hand side on both mobile and desktop views. I think this is more easily handled in the App component.